cpp/src/Slice/Scanner.l

```diff
@@ -472,7 +472,7 @@ floating_literal    (({fractional_constant}{exponent_part}?)|((\+|-)?{dec}+{expo
     StringTokPtr str = make_shared<StringTok>();
     str->literal = yytext;
     *yylval = str;
-    return BAD_TOKEN;
```

Comment:
This is where we now treat unquoted string literals as-if they were quoted.

If you look at the lines just above you this you'll see it only affects `METADATA`, and we still emit an error message.
It just simplifies the grammar rules if we return a normal string-literal instead of a bad-token.